queries/places.jinja2

```diff
@@ -70,9 +74,13 @@ SELECT
     NULL AS population,
     NULL AS capital,
     NULL AS state_capital,
+    n_photos,
```

Comment:
Yup, it was meant to be "number of". It will be exposed currently because I'm not dropping it, but it's easy to drop it downstream if that's what we wanted. I figured I would include it since we're including area, at least in the interim for testing purposes.

I like the idea of adding an `mz_` prefix to it if we're going to drop it from the output.

If we want to keep it, is it better to call this `num_photos` instead? Or just `photos` like you mentioned?